app/policies/kudo_policy.rb

```diff
@@ -0,0 +1,9 @@
+class KudoPolicy < ApplicationPolicy
+  def update?
+    true if Time.current <= record.created_at.advance(minutes: 5)
+  end
+
+  def destroy?
+    true if Time.current <= record.created_at.advance(minutes: 5)
+  end
```

Comment:
I suggest checking here if `current_employee` is also `kudo.giver`. This will allow us not to do two ifs (for correct_employee, and for time creation) but to only check if policy is true.

Reply:
Honestly I thought about it but I found one problem which made me to leave these checks separately. Namely now the application has three types of links in kudos index page: (edit, destroy), (in order to match AC in this task -> disabled edit, destroy) and (unauthorized). First type of our links appear when current employee is a kudo.giver and current time is not 5 minutes after sent a kudo. Second types similarly appear when current employee is a kudo giver but current time is 5 minutes after kudo creation. Third types in turn appears when current employee is not a kudo giver. Thus in my opinion with this division of links types it is simpler to keep these two checks apart from each other. This solution guarrantes me really easy way to handle diffrent kind of links in kudos index page.

Reply:
That's right, I missed this AC. In my application, there isn't any button after "edit time", in yours, there is but disabled.

Reply:
Exactly :) Thanks a lot -> I am really grateful for your review!